### PR TITLE
Filter admin links

### PR DESCRIPTION
**Pull request by barryhughes**
Efforts to remove links to wp-admin screens from the Community Tickets UI.

:ticket: [#89015](https://central.tri.be/issues/89015)

### CODE REVIEW

**Comment by barryhughes** [comment]
Removed what I think were some unneeded changes and recompiled. Ready for review.

File: src/Tribe/Linked_Posts/Chooser_Meta_Box.php [hunk 1]
```diff
@@ -191,24 +189,23 @@ protected function single_post_dropdown( $linked_post_id ) {
 	 */
 	protected function edit_post_link( $linked_post_id ) {
 		$linked_post_pto = get_post_type_object( $this->post_type );
-		if (
+		$can_edit_others_posts = (
```
Comment (borkweb) [inline src/Tribe/Linked_Posts/Chooser_Meta_Box.php]: Perhaps this variable should be: `$cannot_edit_others_posts`?

File: src/resources/js/events-admin.js [hunk 2]
```diff
@@ -694,7 +694,7 @@ jQuery( document ).ready( function( $ ) {
 	//show state/province input based on first option in countries list, or based on user input of country
 	$( 'body' ).on( 'change', '#EventCountry', function () {
 		var $country        = $( this );
-			$container      = $country.parents( 'div.eventForm' ).eq( 0 ),
+		$container      = $country.parents( 'div.eventForm' ).eq( 0 ),
```
Comment (borkweb) [inline src/resources/js/events-admin.js]: Alignment is weird on this line.

File: src/Tribe/Linked_Posts/Chooser_Meta_Box.php (hunk 1 above)
Comment (barryhughes) [reply src/Tribe/Linked_Posts/Chooser_Meta_Box.php]: Ahh ... yes. I must have been mixing up _'can'_ and _'cannot'_ - will fix ;-)

File: src/resources/js/events-admin.js (hunk 2 above)
Comment (barryhughes) [reply src/resources/js/events-admin.js]: Yeah scoping is crappy too, now you point it out. Will fix.

**Pull request merged**